### PR TITLE
Rn 360

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: react/features/video-quality/middleware.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            const { conference, maxReceiverVideoQuality } = state['features/base/conference'];
+            const { resolution } = state['features/base/config'];
+
+            if (typeof resolution !== 'undefined') {
+                if (resolution < VIDEO_QUALITY_LEVELS.LOW) {
+                    logger.error(`Invalid frame height for video quality - ${resolution}`);
+
+                    break;
+                }
+                conference.setReceiverVideoConstraint(Math.min(resolution, maxReceiverVideoQuality));
```
Comment (jallamsetty1) [inline react/features/video-quality/middleware.js:31]: Don't we want to set the senderVideoConstraint as well ? I thought we wanted to send up to 360p on the mobile clients.

Comment (saghul) [reply react/features/video-quality/middleware.js:31]: That is done via config. Setting the sender constraint doesn’t work on mobile.

**Comment by saghul** [comment]
Thanks for the review @jallamsetty1 ! Can you PTAL again? I had to do some extra adjustments so the value would be persisted properly across tile view changes and the like.

File: react/features/video-quality/middleware.js (hunk 1 above)
Comment (jallamsetty1) [reply react/features/video-quality/middleware.js]: oh right, it works only for p2p case and not the JVB case on RN.

**Comment by paweldomas** [comment]
I'd like to go through it before it gets merged too. I need to understand how all these different flags and config options for resolutions play together.

**Comment by saghul** [comment]
Sure thing, thanks for taking a look @paweldomas ! I did put these commits on a branch for testing, and the latest beta is built with that branch, in order to easily test it out.

**Pull request merged**